static/src/javascripts/projects/commercial/modules/prebid/bid-config.spec.js

```diff
@@ -729,11 +734,27 @@ describe('bids', () => {
     });
 
     test('should include AppNexus directly if in target geolocation', () => {
-        shouldIncludeAppNexus.mockReturnValue(true);
+        shouldIncludeAppNexusAu.mockReturnValue(true);
+        shouldIncludeAppNexusUkRow.mockReturnValue(true);
+        isInAuRegion.mockReturnValue(true);
+        expect(bidders()).toEqual([
+            'ix',
+            'sonobi',
+            'and-au',
+            'and-uk-row',
+            'improvedigital',
+            'xhb',
+            'adyoulike',
+        ]);
+    });
+
+    test('should include AppNexus directly if in target geolocation', () => {
```

Comment:
`should` >> `should exclude AppNexus`?

Test statement doesn't match the outcome here 🙃